dev/source/docs/ros2_pi.rst

```diff
@@ -0,0 +1,96 @@
+.. _ros2-pi:
```

Comment:
```suggestion
.. _ros2_pi:
```

Reply:
Why did you suggest this? None of the other pages match this convention - I was matching the existing hyphen convention. With this changes, it no longer builds.